src/main/java/nextstep/courses/service/CourseService.java

```diff
@@ -62,4 +66,22 @@ public void registerSession(Long sessionId, NsUser nsUser) {
                 .validateRegister(nsUser);
         studentRepository.registerSession(registered.getId(), nsUser.getId());
     }
+
+    @Transactional
+    public void approveSessionRegister(Long sessionId, NsUser nsUser) {
+        sessionRepository.findById(sessionId)
+                .initUsers(studentRepository.findBySessionId(sessionId))
+                .initCandidates(candidateRepository.findBySessionId(sessionId))
+                .validateApprove(nsUser);
```

Comment:
```suggestion
        Student student = sessionRepository.findById(sessionId)
                .initUsers(studentRepository.findBySessionId(sessionId))
                .initCandidates(candidateRepository.findBySessionId(sessionId))
                .approve(nsUser);
```

유효성 체크만 하니까 로직을 수행하는 느낌이 좀 덜하다
승인 가능하다면 Student를 반환하고, 반환한 Student를 DB에 저장하는 구조는 어떨까?